<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageFileSystemProjectImport</class>
 <widget class="QWizardPage" name="WizardPageFileSystemProjectImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Import Projects from File System</string>
  </property>
  <property name="subTitle">
   <string>This wizard analyzes the content of your folder to find projects and import them.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>From Directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditDestDirectory"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>connectionString</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectAll">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeselectAll">
       <property name="text">
        <string>Deselect All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxOverwriteData">
        <property name="text">
         <string>Overwrite existing files without waring</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonBrowse</sender>
   <signal>released()</signal>
   <receiver>WizardPageFileSystemProjectImport</receiver>
   <slot>browseForImportDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDestDirectory</sender>
   <signal>textEdited(QString)</signal>
   <receiver>WizardPageFileSystemProjectImport</receiver>
   <slot>onImportDirectoryChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelectAll</sender>
   <signal>released()</signal>
   <receiver>WizardPageFileSystemProjectImport</receiver>
   <slot>setAllProjectsChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDeselectAll</sender>
   <signal>released()</signal>
   <receiver>WizardPageFileSystemProjectImport</receiver>
   <slot>setAllProjectsUnchecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseForImportDirectory()</slot>
  <slot>onImportDirectoryChanged(QString)</slot>
  <slot>setAllProjectsChecked()</slot>
  <slot>setAllProjectsUnchecked()</slot>
 </slots>
</ui>
